zebra/zebra_srv6.h

```diff
@@ -139,6 +142,27 @@ srv6_sid_alloc_mode2str(enum srv6_sid_alloc_mode alloc_mode)
 	}
 }
 
+PREDECL_DLIST(zebra_srv6_sid_client_list);
+
+struct zebra_srv6_sid_client {
+	struct zserv *client;
+
+	struct zebra_srv6_sid_client_list_item item;
+};
+
+DECLARE_DLIST(zebra_srv6_sid_client_list, struct zebra_srv6_sid_client, item);
```

Comment:
does this need to be defined here - it looks like it's only used in the .c file?

Reply:
Thanks for the review.

Yes, this needs to be declared here in `zebra_srv6.h` because it is used in both `zebra_srv6.c` and `zebra_srv6_vty.c`.